solr/solrj/src/java/org/apache/solr/client/solrj/impl/SolrClientBuilder.java

```diff
@@ -34,6 +35,8 @@
   protected boolean useMultiPartPost;
   protected Integer connectionTimeoutMillis = 15000;
   protected Integer socketTimeoutMillis = 120000;
+  protected long retryExpiryTime =
```

Comment:
SolrClientBuilder is for all the Apache HttpClient based SolrClients, but retryExpiryTime is specifically for ZK.  It's wrong to put here.

Reply:
Good catch...